### ISSUE TITLE
Organize list of people

### ISSUE THREAD

**Issue by erikkp** [Eziowrf/projectGedcom#2]
order the list of all people by ID number

**Comment by jaxyyf** [comment]
I don't think this is a problem because prettytable in Python has automatically done this.

**Comment by erikkp** [comment]
I don't think we implemented any form of "sort" function for the people. I think we are just doing it off order in the GEDCOM files which happen to be in order.

**Comment by jaxyyf** [comment]
Emmm, yeah you are right! That's my bad. So is a GEDCOM file always arranged in order?
